cmd/swarmctl/cluster/update.go

```diff
@@ -136,5 +143,6 @@ func init() {
 	updateCmd.Flags().StringSlice("secret", nil, "Secret required to join the cluster")
 	updateCmd.Flags().Int64("taskhistory", 0, "Number of historic task entries to retain per slot or node")
 	updateCmd.Flags().Duration("certexpiry", 24*30*3*time.Hour, "Duration node certificates will be valid for")
+	updateCmd.Flags().StringSlice("external-ca-url", nil, "URL to one or more CFSSL CA certificate signing endpoints")
```

Comment:
There should probably also be an `external-ca-type` to specify the type of external CA, e.g. `cfssl` or `vault`. Also an `external-ca-args` to specify arguments for the API client.